### ISSUE TITLE
Make 'Testnet' more obvious and separated from 'Mainnet'

### ISSUE THREAD

**Issue by colourfreak** [AlphaWallet/alpha-wallet-android#1804]
Clickable prototype (iOS):
https://invis.io/GU10Q3YYSZB7

- You have a toggle to enable Testnets. This toggle disables Mainnet and enable Testnet. And other way around. Only one can be enabled at the time.
- After you hit the toggle, it warns you about what testnets are
- We say that they have no value at all time.
- When you are viewing a Wallet in a Testnet, all Mainnet tokens are hidden.
- You have a special link that on tap shows a modal explaining "Where are my Tokens?"
- You can go to "Switch to Mainnet" which will go to Select Active Networks again

Zeplin:
https://zpl.io/VDgG0Dg

![testnet_flow_android](https://user-images.githubusercontent.com/51817359/114670586-73e85200-9d03-11eb-83b6-98f93f0715dc.jpg)

**Comment by JamesSmartCell** [comment]
@colourfreak Would it be clearer if there was a single switch that throws between mainnets on one side and testnets on the other. Having two switches like this seems a bit odd. Also, if I'm coming back to the wallet after upgrade I'm going to wonder where the testnets are, I have to know to go digging in the settings. 

Could we duplicate the switch or do a program-link in the 'add / hide tokens'? Although I'm not sure if that works.

**Comment by colourfreak** [comment]
> @colourfreak Would it be clearer if there was a single switch that throws between mainnets on one side and testnets on the other. Having two switches like this seems a bit odd. Also, if I'm coming back to the wallet after upgrade I'm going to wonder where the testnets are, I have to know to go digging in the settings.
> 
> Could we duplicate the switch or do a program-link in the 'add / hide tokens'? Although I'm not sure if that works.

I thought about it, but there are few things to consider:
- I don't know any material design component that uses that kind of switch.
- We could have just one toggle with "Testnet", but how many users do know what is the opposite to testnet? I mean, what you need to know is between what you are switching (testnet/mainnet) and in which state you are in. 

<img width="694" alt="Screenshot 2021-04-20 at 12 58 27" src="https://user-images.githubusercontent.com/51817359/115385548-8b7a7b80-a1d8-11eb-983e-976e93824855.png">

For example, when I select "Poland" I know what to expect. And if I disable this, I assume it will show me worldwide results. 
But in our example, very few users will know, that the opposite to "testnet". And if you can't name something, you can't even learn in Google what is this. 

And in regards to Add/Hide tokens - let's leave it for now until we will have Samoa.

**Comment by colourfreak** [comment]
To improve: 
**1. Instead of select buttons, we should always use a checkbox.** 

<img width="882" alt="Screenshot 2021-04-21 at 11 28 06" src="https://user-images.githubusercontent.com/51817359/115531067-a6aac100-a294-11eb-97f3-fa7f75defa98.png">

![2021-04-21 11 25 22](https://user-images.githubusercontent.com/51817359/115531319-ebcef300-a294-11eb-8baa-1df541af6fed.jpg)
![2021-04-21 11 25 29](https://user-images.githubusercontent.com/51817359/115531336-effb1080-a294-11eb-8348-9e39f31fcfcc.jpg)


**2. We are missing a modal**
https://zpl.io/VKdpz4X

not sure what this modal is
![2021-04-21 11 24 53](https://user-images.githubusercontent.com/51817359/115531464-191ba100-a295-11eb-92cd-5c4ddcd128bf.jpg)

3. By default all buttons are "checkbox" button types. When you switch to "Testnet" they become "select" button types. They should not be. Checkouts all time. 

https://user-images.githubusercontent.com/51817359/115532344-fc339d80-a295-11eb-9ba7-3100ca85f167.mp4

**Comment by colourfreak** [comment]
https://web.tresorit.com/l/sUllm#gq9rFgjex17XeU4zMws4Lw

**Comment by ChintanRathod** [comment]
@JamesSmartCell There are 2 changes 
1. For browser tab "home" button to add 
2. Need to add "Network: Ethereum" option and in the next screen need to show a shortcut for "active network selection"

![image](https://user-images.githubusercontent.com/4371780/115547154-92ca8380-a2c3-11eb-832d-ca0336abe4d0.png)
![image](https://user-images.githubusercontent.com/4371780/115547167-98c06480-a2c3-11eb-9cea-5794eddeeb6a.png)

**Comment by colourfreak** [comment]
@ChintanRathod Please disregard what I told you on the call. I spoke with James right after, and we agreed on:

**Browser**
https://zpl.io/Vq7LmA5

The current implementation in the Browser tab (how you switch between networks) is good. But we need to make two small adjustments:

<img width="1719" alt="Screenshot 2021-04-21 at 15 23 49" src="https://user-images.githubusercontent.com/51817359/115561464-e209b780-a2b5-11eb-9396-d0082364acb7.png">

1. Add a modal window when enabling Testnet - this is to stop scammers. Users don't get the difference between Mainnet and Testnet so we have to remind them. 
Modal: https://zpl.io/VDg41kq
2. Change the Title of the screen to "Connect to Network".

----

**Settings**
This requires a bit of work. But we are very close. 
https://github.com/AlphaWallet/alpha-wallet-android/issues/1804#issue-857605286

1. If you go to Settings -> Select Active Networks you will find the checkboxes with network names. You can select as many as you want. 
2. When you switch to Testnet, show a modal: https://zpl.io/VKdpz4X
3. Currently, when you switch to Testnet it also asks you which network you want to reassign a browser to. Don't do this. Users can choose it in a browser, not here in the Settings. 

So to sum up:
**Connect to network** in a Browser is to easily connect Web3.0 browser to the desired network. You don't have to pre-activate that networks first. Instead, just click and it's switched. 

**Select Active Networks** in the Settings tab is for users to decide what should be visible or not in a Wallet.

Does it make sense? @JamesSmartCell
